api/apps/api/src/modules/uploads/upload-limits.ts

```diff
@@ -0,0 +1,23 @@
+import { MulterOptions } from '@nestjs/platform-express/multer/interfaces/multer-options.interface';
+import { AppConfig } from '@marxan-api/utils/config.utils';
+
+export const simpleGeometry = (): MulterOptions['limits'] => ({
+  fileSize: (() =>
+    AppConfig.get<number>('fileUploads.limits.singleGeometry', 1048576))(),
```

Comment:
I'd suggest setting default values in `default.json`, instead of declaring them here (same for the other 2 values below)

Reply:
@ikas the reason I removed those from `json` is pretty simple - due to nature of `AppConfig`, we would anyway need to define the `default` value when `.get`ting value (or it would crash the whole application) - repeating the same value  makes it harder to maintain.

As long as we have a single place to keep the default - together with keeping in mind the issue above, as well as the fact that .json is anyway kept in repository - decision to keep those in code seems to be right. (not to mention its easier to find out defaults navigating in code).

Does this sound fair?

Reply:
yep, that's fine :+1: